src/components/Albums/Albums.js

```diff
@@ -15,12 +16,12 @@ export class AlbumsComponent {
     this.albumRipplePoint = this.mountPoint.querySelectorAll(
       ".albums__card-ripple-effect"
     );
+    this.albumsContainer = this.mountPoint.querySelector(".albums__container");
   }
 
   initMaterial() {
     Array.from(this.albumRipplePoint).forEach(el => {
-      // eslint-disable-next-line no-new
-      new MDCRipple(el);
+      this.albumRipplePoint = new MDCRipple(el);
```

Comment:
why did you do this? what is the purpose to assign each time to this smth in the loop?

Reply:
To assign each album ripple effect.